web/src/components/SecureAdminConsole.jsx

```diff
@@ -207,7 +207,16 @@ class SecureAdminConsole extends React.Component {
     }
   }
 
-  async componentWillMount() {
```

Comment:
Diff looks weird but I moved the logic from `componentWillMount` into `componentDidMount` only changing what is noted